REST-API-Basics/web/src/main/java/com/epam/esm/web/controller/TagController.java

```diff
@@ -0,0 +1,78 @@
+package com.epam.esm.web.controller;
+
+import com.epam.esm.model.entity.Tag;
+import com.epam.esm.service.exception.ServiceException;
+import com.epam.esm.service.service.TagService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+
+@RestController
+@RequestMapping("/tags")
+public class TagController {
+
+    private final TagService tagService;
+
+    @Autowired
+    public TagController(TagService tagService) {
+        this.tagService = tagService;
+    }
+
+    @GetMapping
+    @ResponseStatus(HttpStatus.OK)
+    public List<Tag> getAll() {
+        return tagService.getAll();
+    }
+
+    @GetMapping(value = "/id/{tagId}")
```

Comment:
Not corresponds with REST naming principles, check other endpoints for this controller too.